jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/logging/AnsiLoggerWithFooterTest.java

```diff
@@ -163,28 +171,40 @@ public void testLog_changingFooter() {
 
     Assert.assertEquals(
         Arrays.asList(
-            "\033[1mfooter\033[0m",
-            "\033[1A\033[0J",
-            "\033[1Amessage",
-            "\033[1mfooter\033[0m",
-            "\033[1A\033[0J",
-            "\033[1A\033[1mtwo line\033[0m",
-            "\033[1mfooter\033[0m",
-            "\033[1A\033[1A\033[0J",
-            "\033[1Aanother message",
-            "\033[1mtwo line\033[0m",
-            "\033[1mfooter\033[0m"),
+            "\033[1mfooter\033[0m", // single-line footer in bold
```

Comment:
Does this test make sure the original problem is fixed?

Reply:
In a way, yes for both tests. The old test was testing moving the cursor up one line. Now it tests moving up twice and down once. And tests `accept()` is called twice with right log level (`Level.WARN` in this test).